Assets/Scripts/Internal/DaggerfallBillboard.cs

```diff
@@ -32,55 +27,38 @@ namespace DaggerfallWorkshop
 #endif
     [RequireComponent(typeof(MeshFilter))]
     [RequireComponent(typeof(MeshRenderer))]
-    public class DaggerfallBillboard : MonoBehaviour
+    public class DaggerfallBillboard : Billboard
     {
-        public int FramesPerSecond = 5;     // General FPS
-        public bool OneShot = false;        // Plays animation once then destroys GameObject
-        public bool FaceY = false;          // Billboard should also face camera up/down
-
-        [SerializeField]
-        BillboardSummary summary = new BillboardSummary();
-
-        public int customArchive = 210;
```

Comment:
Please leave `customArchive` and `customRecord` intact. They're used by `DisplayAboutGUI` in DaggerfallBillboardEditor.cs. Removing these breaks the editor script and prevents changing billboard material in editor (billboard editor calls `SetMaterial` when user changes these fields).

The reason nothing appears to use these fields is because the editor abstracts scripts using `serializedObject.FindProperty(name);` which I call from a `Prop` helper.

Reply:
Sorry, that tricked me - I was really sure they were unused vestiges. Have restored them now.